legal-api/src/legal_api/reports/business_document.py

```diff
@@ -565,18 +569,26 @@ def _set_continuation_in_details(self, business: dict):
             continuation_in_filing = continuation_in_filing[0]
             jurisdiction = Jurisdiction.get_continuation_in_jurisdiction(continuation_in_filing.business_id)
 
+            if not jurisdiction:
+                return
+
             # Format country and region
             region_code = jurisdiction.region
             country_code = jurisdiction.country
-            country = pycountry.countries.get(alpha_2=country_code)
-            region = None
-            if region_code and region_code.upper() != 'FEDERAL':
-                region = pycountry.subdivisions.get(code=f'{country_code}-{region_code}')
-            location_jurisdiction = f'{region.name}, {country.name}' if region else country.name
+            location_jurisdiction = 'Not Available'
```

Comment:
Fix for `psycopg2.errors.NotNullViolation) null value in column "country" of relation "jurisdictions"` - Part 2